models/Recipe.model.js

```diff
@@ -1,8 +1,48 @@
 const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
+/*
+  required, enum, default, set, unique, 
+*/
+
 const recipeSchema = new Schema({
-  // TODO: write the schema
+  title: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  level: {
+    type: String,
+    enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef'],
+  },
+  ingredients: {
+    type: [String],
+  },
+  cuisine: {
+    type: String,
+    required: true
+  },
+  dishType: {
+    type: String,
+    enum: ['breakfast', 'main_course', 'soup', 'snack', 'drink', 'dessert', 'other']
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg"
+    //set: value => value.image('https://images.media-allrecipes.com/images/75131.jpg')
+  },
+  duration: {
+    type: Number,
+    min: 0,
+
+  },
+  creator: {
+    type: String,
+  },
+  created: {
+    type: Date,
+    default: new Date
```

Comment:
No estoy seguro de que esto funcione. Es cuestión de probar. En todo caso una alternativa fiable es:

```suggestion
    default: Date.now
```